app/src/main/java/aashrai/android/gettowork/view/activity/SettingsActivity.java

```diff
@@ -49,6 +50,8 @@
   }
 
   private void configureSearch() {
+   search.setCompoundDrawables(null, null,
```

Comment:
Wait if ContextCompat.getDrawable works just fine then we don't need to use VectorDrawableCompat